tools/git-prompt.sh

```diff
@@ -109,7 +109,8 @@ __git_ps1_show_upstream ()
 
 	svn_remote=()
 	# get some config options from git-config
-	local output="$(git config -z --get-regexp '^(svn-remote\..*\.url|bash\.showupstream)$' 2>/dev/null | tr '\0\n' '\n ')"
+	local output
+	output="$(git config -z --get-regexp '^(svn-remote\..*\.url|bash\.showupstream)$' 2>/dev/null | tr '\0\n' '\n ')"
```

Comment:
This file is taken from the upstream git ([`/contrib/completion/git-prompt.sh`](https://github.com/git/git/blob/d9d677b2d8cc5f70499db04e633ba7a400f64cbf/contrib/completion/git-prompt.sh)). We wouldn't like to introduce unneeded diffs so that it makes it easier to later merge the new version from the upstream.

By the way, we do not use the exit status of this line, and also this is the configuration for the interactive session. so `set -e` is irrelevant. For these reasons, we do not need to separate these two lines.

Reply:
I mean I'd like to keep this file untouched because this file is not managed by this project.

Reply:
Thanks, Koichi. Yeah, that definitely makes sense.

Reply:
Thanks!